### DISCUSSION TITLE
Video Streaming like youtube

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11486] [Q&A]
I uploaded the video on admin side below:

![image](https://user-images.githubusercontent.com/5202291/161511440-d9477810-1113-45ee-9496-3c2959266a2d.png)

This video display on front end:

![image](https://user-images.githubusercontent.com/5202291/161513025-273895e0-7d49-475b-97ef-82e361b33502.png)

But our requirement is to show that video like youtube streaming or placeholder video also not showing? like below:

![image](https://user-images.githubusercontent.com/5202291/161513536-a62ac482-38cb-436c-a1f3-42bc6c26e984.png)

**Comment by hishamco** [comment]
You can use any JavaScript video player on the front-end, but if your videos are mp4 the `<video>` tag is enough to render them natively in the browser

**Comment by saleemhaq** [comment]
Any library you suggest So it can act like YouTube video streaming?

**Comment by ns8482e** [comment]
@saleemhaq I guess you need to use alternate to override default template

**Comment by saleemhaq** [comment]
Can you please suggest me any?

**Reply by ns8482e** [reply]
https://www.google.com/search?q=html5+video+player

**Reply by hishamco** [reply]
@saleemhaq many of your inquries needs a simple search, so please try to do some initiative from your side, in case you faced an issue add your question, this way you will help the folks to assist you

**Reply by saleemhaq** [reply]
> @saleemhaq many of your inquries needs a simple search, so please try to do some initiative from your side, in case you faced an issue add your question, this way you will help the folks to assist you

How I used that package: restream-module-orchardcore in razor, I go through the below link:

https://www.buymeacoffee.com/nicolasmaluleke/restream-module-orchardcore

but unable to utilize in razor syntax

**Reply by saleemhaq** [reply]
> https://www.google.com/search?q=html5+video+player

is that good to go: have any experience regarding this: How I used that package: restream-module-orchardcore in razor? syntax there is liquid, I go through the below link:
https://www.buymeacoffee.com/nicolasmaluleke/restream-module-orchardcore
but unable to utilize in razor syntax, please advise @ns8482e

**Reply by ns8482e** [reply]
@saleemhaq  you can ask that question to author of that module in their repo

**Reply by hishamco** [reply]
Seems you asked the same thing [here](https://github.com/OrchardCMS/OrchardCore/discussions/11497), as @ns8482e said please the best to ask the author of the module
